pennylane_lightning/core/simulators/lightning_gpu/bindings/LGPUBindings_nb.hpp

```diff
@@ -69,7 +160,57 @@ void registerBackendSpecificMeasurements(PyClass &) {} // pyclass
  * @param m Nanobind module
  */
 template <class StateVectorT>
-void registerBackendSpecificObservables(nb::module_ &) {} // m
+void registerBackendSpecificObservables(nb::module_ &m) {
+    using PrecisionT =
+        typename StateVectorT::PrecisionT; // Statevector's precision.
+    using ComplexT =
+        typename StateVectorT::ComplexT; // Statevector's complex type.
+    using ParamT = PrecisionT;           // Parameter's data precision
+
+    const std::string bitsize =
+        std::to_string(sizeof(std::complex<PrecisionT>) * 8);
+
+    using ArrayT = nb::ndarray<std::complex<ParamT>, nb::c_contig>;
+
+    std::string class_name;
+
+    class_name = "SparseHamiltonianC" + bitsize;
+    using IdxT = typename SparseHamiltonian<StateVectorT>::IdxT;
```

Comment:
Let's make it `IndexT` consistently over all bindings.
```suggestion
    using IndexT = typename SparseHamiltonian<StateVectorT>::IdxT;
```